unit-testing-best-practices.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Also add other meta tags such as description and keywords. :) More info: https://www.wordstream.com/meta-tags

Reply:
Good idea, I'll do a separate PR and do this for all my existing pages.